code/lib/builder-vite/src/plugins/external-globals-plugin.ts

```diff
@@ -47,9 +48,7 @@ export async function externalGlobalsPlugin(externals: Record<string, string>) {
       }
       const newAlias = mergeAlias([], config.resolve?.alias) as Alias[];
 
-      const cachePath = join(process.cwd(), 'node_modules', '.cache', 'vite-plugin-externals');
-      await ensureDir(cachePath);
-      await emptyDir(cachePath);
+      const cachePath = findCacheDirectory({ name: 'sb-vite-plugin-externals', create: true }) as string;
```

Comment:
I've noticed that you have used this `sb-` prefix notation elsewhere, so I though to unify it here as well

Reply:
Also ... I don't much like the string type coercion, but I don't have a clue what would be the correct way of making sure that this passes type safety… as far as the real implementation goes, this should be at the same safety level as before because if the `undefined` return case is because there might be some problem with access rights, `ensureDir` & `emptyDir` would have those too